core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -1123,7 +1118,6 @@ public String toString() {
       new Builder(Name.MASTER_AUDIT_LOGGING_ENABLED)
           .setDefaultValue(false)
           .setDescription("Set to true to enable file system master audit.")
-          .setIgnoredSiteProperty(true)
```

Comment:
@calvinjia do you know why this was marked as `IgonredSiteProperty`? This property is not really used any where else (like log4j or sh script) but only in Java land. I tried remove this line and audit log still works

Reply:
Does audit logging work if you remove this line and set it through the site properties?